src/clue.js

```diff
@@ -83,12 +83,175 @@ name: Theater
+  age: 26,
+  image: 'http://www.radiotimes.com/uploads/images/Original/111967.jpg',
+  occupation: 'Scientist',
+}
+
+var profPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'Billionare video game designer',
+  age: 22,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg',
+  occupation: 'Designer',
+}
+
+var missScarlet = {
+  firstName: 'Kasandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is an A-list movie star with a dark past',
+  age: 31,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg',
+  occupation: 'Actor',
+}
+
+var mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is from a wealthy family and uses her status and money to earn popularity',
+  age: 36,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+  occupation: 'Socialité',
+}
+
+var mrMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is a former football player who tries to get by on his former glory',
+  age: 62,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg',
+  occupation: 'Retired Football player',
+}
+
+var rope = {
+  name: 'rope',
+  weight: 10,
+}
+var knife = {
+  name: 'knife',
+  weight: 8,
+}
+var candlestick = {
+  name: 'candlestick',
+  weight: 2,
+}
+var dumbbell = {
+  name: 'dumbbell',
+  weight: 30,
+}
+var poison = {
+  name: 'poison',
+  weight: 2,
+}
+var axe = {
+  name: 'axe',
+  weight: 15,
+}
+var bat = {
+  name: 'bat',
+  weight: 13,
+}
+var trophy = {
+  name: 'trophy',
+  weight: 25,
+}
+var pistol = {
+  name: 'pistol',
+  weight: 20,
+}
+
+var dinningRoom = {
+  name: 'Dinning Room',
+}
+var conservatory = {
+  name: 'Conservatory',
+}
+var kitchen = {
+  name: 'Kitchen',
+}
+var study = {
+  name: 'Study',
+}
+var library = {
+  name: 'Library',
+}
+var billiardRoom = {
+  name: 'billiardRoom',
+}
+var lounge = {
+  name: 'Lounge',
+}
+var ballroom = {
+  name: 'Ballroom',
+}
+var hall = {
+  name: 'Hall',
+}
+var spa = {
+  name: 'Spa',
+}
+var livingRoom = {
+  name: 'livingRoom',
+}
+var observatory = {
+  name: 'Observatory',
+}
+var theater = {
+  name: 'Theater',
+}
+var guestHouse = {
+  name: 'guestHouse',
+}
+var patio = {
+  name: 'Patio',
+}
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [dinningRoom, conservatory, kitchen, study, library, billiardRoom, lounge, ballroom, hall, spa, livingRoom, observatory, theater, guestHouse, patio];
+
+
+function randomSelector(arr) {
+  var randomNum = Math.floor(Math.random()*arr.length);
```

Comment:
It is actually the index of the array right? Maybe you could change the name to give it more meaning.